Sources/SBTUITestTunnelClient/SBTUITestTunnelClient.m

```diff
@@ -608,7 +625,11 @@ - (BOOL)userDefaultsResetSuiteName:(NSString *)suiteName;
     if (objectBase64) {
         NSData *objectData = [[NSData alloc] initWithBase64EncodedString:objectBase64 options:0];
         
+        // this can't switch to the non-deprecated NSSecureCoding method because the types aren't known ahead of time
```

Comment:
Open question: since this is ultimately sourced from a plist, should we just pass in all valid plist types and attempt to use `NSSecureCoding`?

Reply:
I would keep this as is for the sake of simplicity, wdyt?

Reply:
Sounds good